.github/workflows/deploy.yml

```
@@ -34,12 +43,22 @@ jobs:
+          path: ./out
+
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Deploy to GitHub Pages
+        id: deployment
+        uses: actions/deploy-pages@v4
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)